Makefile

```diff
@@ -163,7 +162,8 @@ all: default
 
 $(TARGET): $(SQLITE_SRC) $(WRAPPER_SRC)
 	$(CC) -O3 --sysroot=$(SYSROOT) --target=$(TARGET_TRIPLE) $(SQLITE_FLAGS) $(CFLAGS) $(LDFLAGS) -Wl,$(EXPORT_FUNCS) $^ -o $@.wasm
-	# /root/.cargo/bin/fce embed_it -i sqlite3.wasm -w sqlite3.wit
+	/root/.cargo/bin/marine set version --input ./sqlite3.wasm --version 0.7.0
```

Comment:
let's use the shorthanded command options for that:
`/root/.cargo/bin/marine set version -i ./sqlite3.wasm -v 0.7.0`

to make it more similar to the next command invocation

Reply:
Done